### PR TITLE
Cleanup redundant files in charts

### PR DESCRIPTION
**Pull request by Poor12**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Cleanup redundant files in charts

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
None

### CODE REVIEW

**Comment by RainbowMango** [comment]
Do you mean these files are never used and they are redundant with files in `pre-install-job.yaml`?
https://github.com/karmada-io/karmada/blob/master/charts/karmada/templates/pre-install-job.yaml#L91-L125

**Review by XiShanYongYe-Chang — COMMENTED** [review]
The above cm `{{ $name }}-static-resources` should also be redundant.

File: charts/karmada/templates/post-install-job.yaml [hunk 1]
```diff
@@ -2,59 +2,6 @@
 {{- $namespace := include "karmada.namespace" . -}}
 {{- if eq .Values.installMode "host" }}
 {{- if eq .Values.certs.mode "custom" }}
```
Comment (XiShanYongYe-Chang) [inline charts/karmada/templates/post-install-job.yaml:4]: This line also needs to be deleted.

**Comment by Poor12** [comment]
/close 
In favor of #3721

**Pull request closed without merging**